tsconfig.base.json

```diff
@@ -14,6 +14,8 @@
     "strict": true,
     // save information about the project graph on disk
     "incremental": true,
+    // Do not check d.ts files by default
+    "skipLibCheck": true,
```

Comment:
Can we run typescript with `--skipLibCheck` just on bootstrap? I think adding it here will disable lib checks on CI too.

Reply:
That was my previous thought but we cannot use `--skipLibCheck` flag along with `--build` flag on `tsc` (which we use on build refs during bootstrap). So I'm thinking doing the otherway around. Disable it by default but enable it for typechecking. What do you think @spalger ?

Reply:
Sounds good to me